node_package/src/loadingSpinner/components/TitleLoadingSpinner.jsx

```diff
@@ -121,7 +123,9 @@ export function register() {
       }),
       entryTitle: entryAttribute('title'),
       title: widgetAttribute('title', {role: 'loading_spinner'}),
-      subtitle: widgetAttribute('subtitle', {role: 'loading_spinner'})
+      subtitle: widgetAttribute('subtitle', {role: 'loading_spinner'}),
+      invert: widgetAttribute('invert', {role: 'loading_spinner'}),
```

Comment:
If no custom background image has been selected, the background image of the first page is used. It would be nice if we could also use the `invert` attribute of the first page in this case. Otherwise the text on the loading spinner will be hard to read by default if the first page is inverted and uses a light image. 

We could even consider hiding the invert chech box in the loading spinner configuration editor when no custom background image is selected (see [`visibleBinding` and `visbile` options of input views](https://github.com/codevise/pageflow/blob/0c99e4f60703d8e60870ebb3dec6995205e7fee1/app/assets/javascripts/pageflow/ui/views/mixins/input_view.js#L97)).